### ISSUE TITLE
Add Kalamuna Branding

### ISSUE THREAD

**Issue by RobLoach** [kalamuna/pa11y-dashboard#3]

**Comment by andrewmallis** [comment]
logo is really big, missing alt text 
can we use an SVG?
perhaps we can engage Josh?
